docs/hello_nextflow/10_hello_seqera.md

```diff
@@ -1,7 +1,188 @@
+---
+title: Hello Seqera
+description: Get started with Seqera Platform
+---
+
 # Part 9: Hello Seqera
 
-TODO
+So far we've been running Nextflow workflows on our local machine using the command line interface.
+In this section, we'll introduce you to Seqera Platform, a powerful cloud-based platform for running, monitoring, and sharing Nextflow workflows.
 
----
+!!! tip
+
+    [Sign up](https://cloud.seqera.io/) to try Seqera for free or request a [demo](https://seqera.io/demo/) for deployments in your own on-premise or cloud environment.
+
+You'll learn how to use Seqera Platform within Nextflow CLI, the Seqera Platform GUI, and the API.
+
+You can use Seqera Platform via either the **CLI**, through the **online GUI** or through the **API**.
+
+## 1. Use Seqera Platform to capture and monitor Nextflow jobs launched from the CLI
+
+--8<-- "./seqera/01_run_with_cli.md"
+
+## Online GUI
+
+To run using the GUI, there are three main steps:
+
+1. Create an account and login into Seqera Platform, available free of charge, at [cloud.seqera.io](https://cloud.seqera.io).
+2. Create and configure a new [compute environment](https://docs.seqera.io/platform/24.1/compute-envs/overview).
+3. Start [launching pipelines](https://docs.seqera.io/platform/24.1/launch/launchpad#launchpad).
+
+### Configuring your compute environment
+
+Seqera Platform uses the concept of **Compute Environments** to define the execution platform where a workflow will run.
+
+It supports the launching of workflows into a growing number of **cloud** and **on-premise** infrastructures.
+
+![Compute environments](img/compute_env_platforms.png)
+
+Each compute environment must be pre-configured to enable Seqera Platform to submit tasks. You can read more on how to set up each environment using the links below.
```

Comment:
Include some allusion to Forge (i.e. that we can help with that pre-configuration?)

Reply:
Will be addressed in later iteration.